emails/templates/emails/wrapped_email.html

```diff
@@ -61,12 +61,12 @@
                   {% endif %}
                   {% if recipient_profile.has_premium %}
                     <br />
-                    <img alt="" width="24" style="display: inline-block; margin-bottom: 0px; width: 24px;"  src="{{ SITE_ORIGIN }}/static/images/email-images/tip-purple.png" />
+                    <img alt="" width="24" style="display: inline-block; margin-bottom: 0px; width: 24px;"  src="{{ SITE_ORIGIN }}/images/email-images/tip-purple.png" />
```

Comment:
Hmm, this is interesting, because I'm not sure why this would work. But of course, that's worrying too. Most importantly, the existing URLs _should_ continue to work, because emails we've forwarded before and are already in our users' inboxes are pointing to them.

However, I think I _do_ know why the `/static/` URLs didn't work for you: I'm assuming you didn't run `npm run watch` (or `npm run build`) in the `/frontend/` dir? Because that's the command that places files from `/frontend/public/` into `/frontend/out/`, which is served by WhiteNoise. Could you confirm if running that allows you to re-add `/static/` and, if it does, re-add it? That feels a bit safer to me than relying on something unknown making these URLs work.